From 19fb975b3ab760811eb22820c42fb9abc7e8bf0c Mon Sep 17 00:00:00 2001
From: "Mr. Ivan" <lopesivan.ufrj@gmail.com>
Date: Wed, 22 Apr 2020 03:42:25 -0300
Subject: [PATCH] done: Configura fonte

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 0895a1f..43b9640 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "InconsolataGo Nerd Font Mono:size=31:hinting=true:hintstyle=Regular:minspace=False";
 static int borderpx = 2;

 /*
--
2.7.4

