From bbb8f39a8efe65848fb0cc04aa1fcc28e64aff6e Mon Sep 17 00:00:00 2001
From: "Mr. Ivan" <lopesivan.ufrj@gmail.com>
Date: Fri, 31 Jul 2020 01:02:36 -0300
Subject: [PATCH] Atualiza o arquivo config.def.h

---
 config.def.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.def.h b/config.def.h
index d2b8eea..0c88855 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "InconsolataLGC Nerd Font Mono:size=25:hinting=true:hintstyle=Regular:minspace=False";
-static char *font2 = "Hack Nerd Font Mono:size=25:hinting=true:hintstyle=Regular";
+static char *font = "InconsolataLGC Nerd Font Mono:size=27:hinting=true:hintstyle=Regular:minspace=False";
+static char *font2[] = { "JoyPixels:pixelsize=25:antialias=true:autohint=true" };
 static int borderpx = 2;
 
 /*
-- 
2.17.1

